### PR TITLE
AD Query: fix error message with faulty server

### PR DESCRIPTION
**Pull request by glicht**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/16446

## Description
Better error handling for when the ldap server mis-behaves

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by glicht** [comment]
TDD: build with test that fails with similar scenario to the issue

File: Integrations/Active_Directory_Query/Active_Directory_Query.py [hunk 1]
```diff
@@ -912,9 +912,10 @@ def main():
             demisto.info(message)
             return_error(message)
             return
-    except LDAPSocketOpenError as e:
+    except Exception as e:
         exc_msg = str(e)
-        demisto.info(exc_msg)
+        demisto.info("Failed bind to: {}:{}. {}: {}".format(SERVER_IP, PORT, type(e), exc_msg
```
Comment (anara123) [inline Integrations/Active_Directory_Query/Active_Directory_Query.py]: this should be `demisto.error`

Comment (glicht) [reply Integrations/Active_Directory_Query/Active_Directory_Query.py]: Depends. I am ok with info (what it was also before) as it is in the flow of "test"

**Comment by yardensade** [comment]
Can we close this branch?

**Pull request merged**